tools/docker/Dockerfile

```diff
@@ -264,11 +264,22 @@ SHELL ["/bin/bash", "-c"]
 # Test image that starts up containerd and the naive snapshotter. The default CMD will drop to a bash shell. Overrides
 # to CMD will be provided appended to /bin/bash -c
 FROM firecracker-containerd-unittest as firecracker-containerd-e2etest-naive
+# Install tini init to handle zombies created by our double-forked shims.
+# Unfortunately, we need to do this rather than only use docker's "--init" flag because docker
+# currently puts its default init binary at /dev/init, but we need to bind-mount /dev from
+# the host in order for the snapshotter to work.
+# This should no longer be an issue after github.com/moby/moby #37655 is released.
```

Comment:
This PR: https://github.com/moby/moby/pull/37655/?  I can't quite tell how that's related...

Reply:
Ugh good catch, I meant to link https://github.com/moby/moby/pull/37665, will fix